src/view/App.tsx

```diff
@@ -1,21 +1,39 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
-"use strict";
+// "use strict";
```

Comment:
Unless you plan this to be temporary.  Don't comment, remove. (If temporary, indicate in commented code.
We can always return to previous commits if we need to see what was there.